packages/jaeger-ui/src/components/TracePage/CriticalPath/index.tsx

```diff
@@ -46,38 +46,40 @@ const computeCriticalPath = (
 ): criticalPathSection[] => {
   const currentSpan: Span = spanMap.get(spanId)!;
 
-  const lastFinishingChildSpan = findLastFinishingChildSpan(spanMap, currentSpan, returningChildStartTime);
-  let spanCriticalSection: criticalPathSection;
+  if (currentSpan) {
```

Comment:
I would like to investigate upstream of this call why this could happen, but just a nit on the code:
```suggestion
  if (!currentSpan) {
    return criticalPath;
  }
```

Reply:
That's what comes of re-typing the fix from the minified code in the browser into the real source code. 

Honestly: I'd advise you do check out what is going on here properly. I wanted to provide what I'd done to work around the problem rather than just filing a bug report and walking off. I didn't have the tools available to really work through it properly, though.